src/backend/templates/partials/footer.hbs

```diff
@@ -28,11 +28,16 @@
          {{#if copyright}}
          {{#if copyright.licence}}
          <p>
-         <a href="{{{copyright.licence_url}}}"> <img src="{{{copyright.logo}}}"> </a>
-         &copy; {{copyright.year}}
+        <a href="{{{copyright.licence_url}}}"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/80x15.png" /></a>
```

Comment:
@Lplenka The licence is not fixed for all events. Please don't hard code licence image url.
Please create a sample event https://open-event-dev.herokuapp.com for reference. Please use the URL provided in the API.